src/Components/Display.jsx

```diff
@@ -0,0 +1,38 @@
+import { useState, useEffect } from "react";
+import getArticles from "./api";
+import ArticleCard from "./ArticleCard";
+
+const Display = ({ searchParams }) => {
+	const [allArticles, setAllArticles] = useState([]);
+	const [isError, setIsError] = useState(false);
+	useEffect(() => {
+		setIsError(false);
+		getArticles(searchParams)
+			.then(({ data }) => {
+                console.log(data)
+				let articlesDisplayed = data.articles;
+				setAllArticles(articlesDisplayed);
+			})
+			.catch(() => {
+				setIsError(true);
+			});
+	}, [searchParams]);
+	if (isError) {
```

Comment:
Nice! Can we do the same with a loading state so the user get some visual feedback if a request is in progress?